### ISSUE TITLE
Record numbers cause test failures in test_spec_remote

### ISSUE THREAD

**Issue by mkelley** [NASA-Planetary-Science/sbpy#123]
@migueldvb @gianninapr The target record numbers in `test_spec_remote.py` are causing failures:
```
vega/tests/['DXREAD: requested IOBJ=   900918 is out of bounds, no action taken.\r', 'Record boundaries:                         ...001-90004391                         \r', 'If an SPK ID, look-up with "DES=   900918;"                        \r', ...]
```
Because record numbers can change at any time, I recommending changing these to designations.

**Comment by gianninapr** [comment]
Thanks, I'll change it to designations ASAP

**Comment by migueldvb** [comment]
It seems that replacing `'900918'` by `'103P'` in lines 24 and 63 of `test_spec_remote.py` file fixes the failures.

**Comment by migueldvb** [comment]
@gianninapr could you update the comet in the spec remote tests?

**Comment by gianninapr** [comment]
I can do that

**Comment by mkelley** [comment]
Fixed by #133
